driver/src/main/scala/com/stratio/crossdata/driver/JavaDriver.scala

```diff
@@ -28,21 +28,21 @@ object JavaDriver {
   /**
    * database can be empty ("")
    */
-  case class TableName(tableName: String, database: String)
+
 }
 
-class JavaDriver(properties: java.util.Map[String, ConfigValue]) {
+class JavaDriver(properties: java.util.Map[String, ConfigValue], flattenTables: Boolean) {
 
-  def this(serverHosts: java.util.List[String]) =
-    this(Map(DriverConfigHosts -> ConfigValueFactory.fromAnyRef(serverHosts)))
+  def this(serverHosts: java.util.List[String], flattenTables: Boolean) =
+    this(Map(DriverConfigHosts -> ConfigValueFactory.fromAnyRef(serverHosts)),flattenTables)
 
-  def this() = this(Map.empty[String, ConfigValue])
+  def this(flattenTables: Boolean) = this(Map.empty[String, ConfigValue],flattenTables)
 
-  import JavaDriver._
+  def this() = this (Map.empty[String, ConfigValue],false)
```

Comment:
I think that you must change this line for this:

final val DefaultFlattenTables = false
def this() = this (DefaultFlattenTables)